src/performance/perf.ts

```diff
@@ -48,7 +48,7 @@ class PagePerf {
         // Synchronous load resources in the page
         resTime: parseInt(String(timing.loadEventStart - timing.domContentLoadedEventEnd), 10),
         // Only valid for HTTPS
-        sslTime: location.protocol.includes('https') ?
+        sslTime: location.protocol === 'https' && timing.secureConnectionStart > 0 ?
```

Comment:
![image](https://user-images.githubusercontent.com/26432832/98461424-eef28d00-21e6-11eb-904b-5949e133f8d9.png)

@qinhang3  It looks like `https:`, please confirm :)

Reply:
You are right.. 😊